src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -545,26 +556,41 @@ public void WriteDatabaseRecord(DatabaseRecord databaseRecord, SmugglerProgressB
                     progress.ConflictSolverConfigUpdated = true;
                 }
 
-                if (currentDatabaseRecord?.PeriodicBackups.Count == 0 &&
-                    databaseRecord?.PeriodicBackups.Count > 0)
+                if (databaseRecord?.PeriodicBackups.Count > 0)
                 {
                     if (_log.IsInfoEnabled)
                         _log.Info("Configuring periodic backups configuration from smuggler");
                     foreach (var backupConfig in databaseRecord.PeriodicBackups)
                     {
+                        currentDatabaseRecord?.PeriodicBackups.ForEach(x =>
```

Comment:
If the `currentDatabaseRecord` is null, this won't get executed at all, which I assume isn't intended.
In that case, won't we want to add _all_ of them?